aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Repository.cs

```diff
@@ -0,0 +1,106 @@
+using Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace Repository;
+
+
+class TodosRepo {
```

Comment:
Nit:

```suggestion
public class TodosRepo
{
```

I'm pretty sure `dotnet format` will catch this though.